lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -29,19 +29,18 @@ def search(query, language: nil, replacements: {}, subauth: nil)
         language ||= search_config.language
         url = Qa::LinkedData::AuthorityUrlService.build_url(action_config: search_config, action: :search, action_request: query, substitutions: replacements, subauthority: subauth)
         Rails.logger.info "QA Linked Data search url: #{url}"
-        graph = load_graph(url: url, language: language)
-        parse_search_authority_response(graph)
+        load_graph(url: url, language: language)
+        parse_search_authority_response
       end
 
       private
 
         def load_graph(url:, language:)
-          graph_service = Qa::LinkedData::GraphService.new(url: url)
-          graph_service.filter(language: language, remove_blanknode_subjects: true)
-          graph_service.graph
+          @graph = Qa::LinkedData::GraphService.load_graph(url: url)
+          @graph = Qa::LinkedData::GraphService.filter(graph: @graph, language: language) unless language.blank?
```

Comment:
I think this will introduce a bug, becuase you are no longer filtering blank nodes

Reply:
Looks like you might be missing a test